darwin/DarwinProcessList.c

```diff
@@ -18,6 +18,55 @@ in the source distribution for its full text.
 #include <sys/mman.h>
 #include <utmpx.h>
 #include <err.h>
+#include <sys/sysctl.h>
+#include <stdbool.h>
+
+
+void GetKernelVersion(int ** major, int ** minor, int ** internal) {
```

Comment:
Is the `**` needed here? I may be wrong, but it seems like you only need `int *major` etc. here if you change the bottom lines to be `*major = cMajor;` etc. and the variable declarations to be `int maj, min, inter;`.

Reply:
If I want to use pointers it's needed or the value won't be set.

Since the size of a pointer is the same or bigger than an int, I could use simple pointer.

Reply:
Fixing this.

Reply:
@mc10 should be fixed